portability-spi-transfer/src/test/java/org/datatransferproject/spi/transfer/provider/converter/MediaImporterDecoratorTest.java

```diff
@@ -0,0 +1,86 @@
+package org.datatransferproject.spi.transfer.provider.converter;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.datatransferproject.spi.transfer.provider.ImportResult;
+import org.datatransferproject.spi.transfer.provider.Importer;
+import org.datatransferproject.types.common.models.media.MediaAlbum;
+import org.datatransferproject.types.common.models.media.MediaContainerResource;
+import org.datatransferproject.types.common.models.photos.PhotoModel;
+import org.datatransferproject.types.common.models.photos.PhotosContainerResource;
+import org.datatransferproject.types.common.models.videos.VideoModel;
+import org.datatransferproject.types.common.models.videos.VideosContainerResource;
+import org.datatransferproject.types.transfer.auth.AuthData;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MediaImporterDecoratorTest {
+
+  private MediaImporterDecorator<AuthData> mediaImporter;
+  private List<MediaAlbum> albums;
+  private List<PhotoModel> photos;
+  private List<VideoModel> videos;
+  private Importer<AuthData, PhotosContainerResource> photosImporter;
+  private Importer<AuthData, VideosContainerResource> videosImporter;
+
+  @BeforeEach
+  public void setUp() {
+    photosImporter = (id, ex, ad, data) -> ImportResult.OK;
+    videosImporter = (id, ex, ad, data) -> ImportResult.OK;
+    mediaImporter = new MediaImporterDecorator<>(photosImporter, videosImporter);
+
+    albums = List.of(new MediaAlbum("album1", "name1", "desc1"),
```

Comment:
(optional nitpick) suggestion to make these lines consistent with your others below

```suggestion
    albums = List.of(
        new MediaAlbum("album1", "name1", "desc1"),
```